pkgs/build-support/docker/default.nix

```diff
@@ -541,6 +550,7 @@ rec {
           pure = writeText "${baseName}-config.json" (builtins.toJSON {
             inherit created config;
             architecture = defaultArch;
+            variant = defaultVariant;
```

Comment:
Why isn't this `inherit variant`?

Reply:
Why not, but I don't think I can do this without giving the user the ability to set its own CPU variant in the `buildImage` function :man_shrugging: